financial-connections/src/main/java/com/stripe/android/financialconnections/features/manualentry/ManualEntryInputValidator.kt

```diff
@@ -0,0 +1,50 @@
+package com.stripe.android.financialconnections.features.manualentry
+
+import com.stripe.android.financialconnections.R
+
+internal object ManualEntryInputValidator {
```

Comment:
@kgaidis-stripe grouped all validation and how each error maps to its corresponding error message in this object, in case you find it useful for iOS : )